### PR TITLE
Fix helper

### PR DESCRIPTION
**Pull request by rodinux**
## Problem

- *fix issue #80*

## Solution

- *keep command helper v2*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -24,7 +24,7 @@ upgrade_type=$(ynh_check_app_version_changed)
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
 
-if ynh_app_upgrading_from_version_before 1.3.0~ynh1; then
+ynh_compare_current_package_version --comparison lt --version 1.3.0~ynh1; then
```
Comment (fflorent) [inline scripts/upgrade:27]: Le mot-clef `if` est manquant

```suggestion
if ynh_compare_current_package_version --comparison lt --version 1.3.0~ynh1; then
```

Comment (rodinux) [reply scripts/upgrade]: pffffu , je dois être fatigué ! merci

**Comment by rodinux** [comment]
@OniriCorpe, I push this on master, are you all right ?

**Pull request merged**